web/src/main/angular/src/app/core/components/call-tree/call-tree.component.css

```
@@ -28,6 +28,10 @@
     background-color: var(--background-default);
 }
 
+::ng-deep #call-tree-table .ag-row-exception {
+    background-color: #fff1f1;
```

Please use css-variables for colors.
```suggestion
    background-color: var(--background-row-danger);
```